applications/substrate-identity-directory.md

```diff
@@ -87,8 +84,7 @@ We have experience in several private and open source projects aswell as project
 | 2. | Design an individual identity page | Create design mockups in Figma. |
 | 3. | Implement list page | Build UI components in VueJS used in the identity list page. | 
 | 4. | Implement individual identity page | Build UI components in VueJS used in the individual identity page. 
-| 5. | Implement query logic for identity data | Implement query logic which will be used to retrieve identity data. |  
-| 6. | Query identity activities | Implement query logic which will be used to retrieve activities indicative of a member’s reputation. |  
```

Comment:
Could you move this to a different milestone instead, together with the treasury and governance modules? Feel free to distribute the price across the two milestones as you think fit.

Reply:
I have created Milestone 3 out of that deliverable. @alxs

Reply:
To clarify, I meant you may split the price of this milestone (which previously contained all deliverables) between this and the new milestone.